tests/kafkatest/services/zookeeper.py

```diff
@@ -23,6 +23,7 @@
 
 
 class ZookeeperService(Service):
+    KAFKA_OPTS = ""
```

Comment:
this probably should be an instance attribute, and not a class attribute - 

for example, if there are multiple zk clusters (as with mirrormaker tests), every instance of ZookeeperService will see any change to `ZookeeperService.KAFKA_OPTS` 

Also, since it is intended to be externally modified/set, it shouldn't look like a constant